webclient/webclient.js

```diff
@@ -168,8 +168,9 @@ var WebClient = {
         "feature_set",
         "room_chat_history",
         "client_warnings",
-        "2.6.1_min_version",
         "websocket",
+        "2.6.1_min_version",
+        "2.7.0_min_version",
         /* unimplemented features */
```

Comment:
Unimplemented features? hmm?
We have forgot pw, and idle client timer?

Reply:
I don't get the question, what do you mean?

Reply:
It looks like the list is somehow separated by a comment.
But the divider makes no sense to me:
```
      "clientfeatures" : [
        "client_id",
        "client_ver",
        "feature_set",
        "room_chat_history",
        "client_warnings",
        /* unimplemented features */
        "forgot_password",
        "idle_client",
        "mod_log_lookup",
        "user_ban_history"
]
```

See https://github.com/Cockatrice/Cockatrice/blob/master/webclient/webclient.js#L165-L176

Reply:
Scrape that, it's the webclient.... 🙈

Reply:
The webclient only implements the features before the comment.
The others are still included only to be able to connect to servers requiring/declaring them.